tests/includes/PdoDatabaseTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/******************************************************************************
+ * Wikipedia Account Creation Assistance tool                                 *
+ *                                                                            *
+ * All code in this file is released into the public domain by the ACC        *
+ * Development Team. Please see team.json for a list of contributors.         *
+ ******************************************************************************/
+
+/**
+ * Created by PhpStorm.
+ * User: Matthew
+ * Date: 12/24/16
+ * Time: 00:52
+ */
+
+namespace Waca\Tests;
+
+use Waca\Exceptions\EnvironmentException;
+use Waca\PdoDatabase;
+
+class PdoDatabaseTest extends \PHPUnit_Framework_TestCase
+{
+	/** @var PdoDatabase */
+	private $pdb;
+
+	public function setUp()
+	{
+        $this->markTestIncomplete("Mocking issues.");
+		$this->pdb = $this->getMock("PdoDatabase", array('get_database', 'getDatabaseConnection','get_arguments'));
```

Comment:
4 spaces please, not tabs.

![image](https://cloud.githubusercontent.com/assets/722710/23096498/dbf2a948-f615-11e6-863d-7d7a68597dd3.png)